services/pom.xml

```diff
@@ -222,6 +222,13 @@
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-model-storage-private</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
Could this be removed as it is not needed in `services` module now?

Reply:
Fixed in [272066b](https://github.com/keycloak/keycloak/pull/26484/commits/272066b92015fcd415c9f5c5af75fa34e2c66dc5)